models/api-models.js

```diff
@@ -1,7 +1,14 @@
 const db = require("../db/connection");
+const fs = require("fs/promises");
 
 exports.selectTopics = () => {
   return db.query("SELECT * FROM topics;").then(({ rows }) => {
     return rows;
   });
 };
+
+exports.getApiEndpoints = () => {
```

Comment:
I think it would be better for this to be included in your controller, since models are for interacting with your database and this is just accessing a file, it's definitely doable in just the controller